todo_app/templates/index.html

```diff
@@ -12,7 +12,23 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+		{% for todolist in todolists %}
+			<li>{{todolist.id}}        {{todolist.title}}          {{todolist.status}}</li>
```

Comment:
HTML doesn't care about excess whitespace. If you want more space between the different fields, you need "no break spaces" (`&nbsp;`) or the better way is to put them inside separate elements (e.g. `<span class="px-2">{{todolist.title}}</span>`)